templates/vue-demo-store/components/checkout/CheckoutCartItem.vue

```diff
@@ -5,13 +5,15 @@ const props = withDefaults(
   defineProps<{
     cartItem: Schemas["LineItem"];
     maxQty?: number;
+    // eslint-disable-next-line vue/require-default-prop
+    controller?: ReturnType<typeof useModal>;
```

Comment:
This rule exists for a reason :) Disabling that you're silencing linter but causing runtime error. For example in checkout clicking it is causing error:
![image](https://github.com/shopware/frontends/assets/13100280/d54b2998-b213-4f53-a13d-16ff89d43a51)

I'll propose a different solution in summary

Reply:
It is not required now.